scripts/functions/requirements/osx_port

```diff
@@ -144,7 +144,7 @@ requirements_osx_port_after()
   [[ -n "${port_root}" && -x "${port_root}"  ]] && # verify binary and
   __rvm_readlink_deep port_root                    # resolve any links to point to the final target
   [[ -n "${port_root}" && -x "${port_root}"  ]] && # verify binary and
-  port_root="${port_root%%*(/)bin*(/)port}"        # get    root
+  port_root="${port_root%%/bin/port}"              # get    root
```

Comment:
the `*(/)` was supposed to match duplicated `/` in links... but his version should work just fine in most cases ;)